CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # nf-core/mhcquant: Changelog
 
+## v2.0.0 nf-core/mhcquant "Steel Beagle" - 2021/05/20
+
+### `Added`
+
+DSL1 to DSL2 conversion
+
+    - Different processes based on a unique step in the pipeline
+    - Inclusion of one sub workflow: refine fdr on predicted subset
+    - The process: openms_cometadapter includes commented lines (which could be used as a reference for future module development)
+    - MHCquant pipeline is ran from workflows/mhcquant.nf instead of main
+
+### `Fixed`
+
+### `Dependencies`
+
+### `Deprecated`
+
```

Comment:
You can also remove the lightspeed hummingbird and merge it with Steel Beagle, as it was never released :)

Reply:
The light speed hummingbird is removed